app/layout.tsx

```
@@ -14,19 +32,36 @@ export default function RootLayout({
-      </body>
-    </html>
+    <ClerkProvider>
+      <html lang="en" className="dark">
+        <body className="font-sans bg-bg-primary text-white">
+          <AnimatedBackground variant="particles" intensity="medium" />
+
+          {/* Auth UI - Top Right Corner */}
+          <div className="fixed top-4 right-4 z-50">
+            <SignedOut>
+              <SignInButton mode="modal">
+                <button className="px-4 py-2 bg-cyan-500/20 border border-cyan-500 text-cyan-400 rounded hover:bg-cyan-500/30 transition-colors shadow-lg shadow-cyan-500/20">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)